### PR TITLE
Update "master" branch references to "main".

### PR DESCRIPTION
**Pull request by denyeart**
Update "master" branch references to "main".

Signed-off-by: David Enyeart <enyeart@us.ibm.com>

### CODE REVIEW

File: docs/source/international_languages.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ In this topic, we're going to cover:
...
 languages. For example:
 
-* [Chinese](https://github.com/hyperledger/fabric-docs-i18n/tree/master/docs/locale/zh_CN)
-* [Malayalam](https://github.com/hyperledger/fabric-docs-i18n/tree/master/docs/locale/ml_IN)
-* [Brazilian Portuguese](https://github.com/hyperledger/fabric-docs-i18n/tree/master/docs/locale/pt_BR)
-* [Japanese](https://github.com/hyperledger/fabric-docs-i18n/tree/master/docs/locale/ja_JP)
+* [Chinese](https://github.com/hyperledger/fabric-docs-i18n/tree/main/docs/locale/zh_CN)
+* [Malayalam](https://github.com/hyperledger/fabric-docs-i18n/tree/main/docs/locale/ml_IN)
+* [Brazilian Portuguese](https://github.com/hyperledger/fabric-docs-i18n/tree/main/docs/locale/pt_BR)
+* [Japanese](https://github.com/hyperledger/fabric-docs-i18n/tree/main/docs/locale/ja_JP)
```
Comment (lindluni) [inline docs/source/international_languages.md:23]: Do you want to keep this as main. The docs translations are actually happening on the `release-2.2` branch over there, so the `master` branch and future `main` don't contain the vast majority of the translations, and dont contain any new translations for these languages that have been done in the last 8 months. Perhaps we should just make is `release-2.2` as that is the default branch in this repo

**Review by lindluni — APPROVED** [review]

**Review by ryjones — APPROVED** [review]

File: docs/source/international_languages.md (hunk 1 above)
Comment (denyeart) [reply docs/source/international_languages.md:23]: good point, i think that is worth doing, but let's merge this first.

**Comment by WaleedMortaja** [comment]
I think this should be backported to earlier branches.

**Comment by WaleedMortaja** [comment]
> I think this should be backported to earlier branches.

@denyeart @ryjones

**Comment by denyeart** [comment]
@WaleedMortaja it is in release-2.4 as well. I think the next most important one to cherry-pick to is release-2.2. Feel free to open a PR for release-2.2.

**Comment by WaleedMortaja** [comment]
@Mergifyio backport release-2.2

**Comment by WaleedMortaja** [comment]
@denyeart Seems that I am not allowed to use mergifyio in this context. You can request that from the bot or I will cherry-pick it later (mostly tomorrow), whatever comes first.

**Pull request merged**